settings.yaml

```diff
@@ -9,3 +9,9 @@ excluded_classes:
   type: list
   list_type: compact
   default: "onebox"
+regex_substitute_or_character:
```

Comment:
@md-misko I don't think this approach would work, see my other comment. If you need this to be configurable, than I would propose to have two settings called `leftwordboundary` and rightwordboundary` where you set a list of boundary characters directly (with the default list as they are now). In that case you have full control and don't need the `underscore_as_word_boundary` setting. What do you think?